### PR TITLE
Update close-your-account.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the navigation as it is "Delete tenant" and not directory any more.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3268

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/close-your-account.md [hunk 1]
```diff
@@ -85,10 +85,10 @@ You can also <a href="https://docs.microsoft.com/azure/active-directory/authenti
 
 2. Select **Azure Active Directory**.
 
-3. Switch to the directory you want to delete.
+3. Switch to the organization you want to delete.
 
-4. Select **Delete directory**.
+4. Select **Delete Tenant**.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/close-your-account.md]:
```suggestion
4. Select **Delete tenant**.
```

Comment (JohanFreelancer9) [reply microsoft-365/commerce/close-your-account.md]: Not sure about this capitalization. Please confirm that it is correct.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/commerce/close-your-account.md (hunk 1 above)
Comment (grzegorz-silarski) [reply microsoft-365/commerce/close-your-account.md]: @JohanFreelancer9 I hope the screenshot helps here: Screenshot 2020-10-27 at 08 21 03

File: microsoft-365/commerce/close-your-account.md [hunk 2]
```diff
@@ -85,10 +85,10 @@ You can also <a href="https://docs.microsoft.com/azure/active-directory/authenti
...
 2. Select **Azure Active Directory**.
 
-3. Switch to the directory you want to delete.
+3. Switch to the organization you want to delete.
 
-4. Select **Delete directory**.
+4. Select **Delete tenant**.
 
-5. If your directory fails one or more checks, you see a link to more information on how to pass the checks. After you pass all checks, select **Delete** to complete the process.
+5. If your Organization fails one or more checks, you see a link to more information on how to pass the checks. After you pass all checks, select **Delete** to complete the process.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/close-your-account.md]:
```suggestion
5. If your organization fails one or more checks, you see a link to more information on how to pass the checks. After you pass all checks, select **Delete** to complete the process.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba - apologies, one more.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**